client/src/components/Header.js

```diff
@@ -1,10 +1,78 @@
 import Nav from './Nav';
 
+import logo from '../assets/header/trivial-logo.svg';
+import line from '../assets/header/line.svg';
+import horizontalArrow from '../assets/header/arrow-swap-horizontal.svg';
+import notification from '../assets/header/notification.svg';
+import messages from '../assets/header/messages.svg'
+import messagesDot from '../assets/header/messages-dot.svg'
+import services from '../assets/header/services.svg';
+import avatar from '../assets/header/avatar.png';
+import arrowDropDown from '../assets/header/arrow-drop-down.svg';
+
 const Header = () => {
+    let userFirstName = 'Eric';
```

Comment:
Can we make this dynamic and instead show email's local-part(before '@')?

Reply:
Yes, we can. Let's discuss this idea in our daily meeting.